File: Algorithms/Sorts/Python/BubbleSort.py
# Bubble sort in Python
def bubbleSort(array):
  
    for i in range(len(array)):
        for j in range(0, len(array) - i - 1):
            if array[j] > array[j + 1]:
                temp = array[j]
                array[j] = array[j+1]
                array[j+1] = temp
    return array

print('Sorted Array in Ascending Order:')
print(bubbleSort([add_any_number])
)

File: Algorithms/Sorts/JavaScript/BubbleSort.js
function BubbleSort(arr)
{
    let i, j, temp;
    for(i=0; i<arr.length; i++)
    {
        for(j=0; j < arr.length - i - 1; j++)
        {
            if (arr[j] > arr[j + 1])
            {
                temp = arr[j];
                arr[j] = arr[j + 1];
                arr[j + 1] = temp;
            }
        }
    }

    return arr;
}

console.log(BubbleSort([add_any_number]))
